services/testService/index.js

```diff
@@ -84,3 +59,30 @@ function emitEvent(server, role, session) {
             throw error;
         });
 }
+
+exports.$oninitService = {

```

Comment:
For me it doesn't make sense to have documentation for this endpoint, but we can keep it.

Reply:
Removed